inspirehep/modules/forms/static/js/forms/experiments_typeahead.js

```diff
@@ -32,12 +32,14 @@ define([
       remote: {
         url: '/api/experiments?q=experimentautocomplete:%QUERY*',
         filter: function(response) {
-          return $.map(response.hits.hits, function(el) { return el });
-        }
-      },
+          return $.map(response.hits.hits, function(el) { return el }).sort(function(x, y) {
```

Comment:
`map`ping the identity function on `response.hits.hits` seems completely pointless...